src/main/java/com/scalar/cassy/transferer/AzureBlobFileUploader.java

```diff
@@ -24,6 +24,7 @@
 
 public class AzureBlobFileUploader implements FileUploader {
   private static final Logger logger = LoggerFactory.getLogger(AzureBlobFileUploader.class);
+  private static final int asyncFileUploadLimit = 10;
```

Comment:
```suggestion
  private static final int ASYNC_FILE_UPLOAD_LIMIT = 10;
```
We use all capitals for constants based on the google java guideline.

Reply:
@scalar-boney BTW, AWS uploader uses 3 threads; thus, it uploads 3 files in parallel.
setting `3` might be too conservative?

How long did it take with each 3 and 500? Please give me the performance numbers.

Reply:
@feeblefakie 
> setting 3 might be too conservative?

Yes

> How long did it take with each 3 and 500? Please give me the performance numbers.

### Backup creation with number of files 3 for each asynchronous upload

**Started:** _Wednesday, March 24, 2021 12:46:32.412 PM_
**Finished:** _Wednesday, March 24, 2021 3:40:13.071 PM _
**Backup creation time:**  _3.54 hrs_

![image](https://user-images.githubusercontent.com/58680313/112269178-5fdca200-8cbb-11eb-8b1d-b07ef28a5315.png)
 
|Period(12:46 ~ 15:40)| 99 Percentile (avg) (ms)  | 95 Percentile (avg) (ms) | 90 Percentile (avg) (ms) | RPS (avg) (reqps) | Failure (avg)(reqps) |
|----------------------------|-------------------|--------------------|--------------------|-------------|---------------|
|24 Mar 2021 | 843 | 618 | 529 | 489 | 21 | 0 |
| 23 Mar 2021 | 733 | 541 | 467 | 433 | 22 | 0 |
| 22 Mar 2021 | 677 | 522 | 462 | 424 | 22 | 0 |

### Backup creation with number of files 500 for each asynchronous upload

**Started:** _Wednesday, March 24, 2021 11:54:20.524 AM_
**Finished:** _Wednesday, March 24, 2021 12:36:27.437 PM_
**Backup creation time:**  _42 min_

![image](https://user-images.githubusercontent.com/58680313/112254107-5b0bf400-8ca3-11eb-8c5b-ad8be2d32d16.png)

|Period(11:54:00 ~ 12:37:00)| 99 Percentile (avg) (ms) | 95 Percentile (avg) (ms) | 90 Percentile (avg) (ms) | RPS (avg) (reqps) | Failure (avg) (reqps) |
|----------------------------|-------------------|--------------------|--------------------|-------------|---------------|
|24 Mar 2021 | 2506 | 2059 | 1543 | 1247 | 14 | 0 |
| 23 Mar 2021 | 676 | 519 | 467 | 429 | 27 | 0 |
| 22 Mar 2021 | 598 | 511 | 458 | 415 | 24 | 0 |

Reply:
@scalar-boney How about 10?

Reply:
@feeblefakie san,
The above table structure was previously damaged due to the missing of one title partition.
Could please a take look at that?

### Backup creation with number of files 10 for each asynchronous upload

**Started:** _Tuesday, March 23, 2021 8:18:14.425 PM_
**Finished:** _Tuesday, March 23, 2021 10:33:48.970 PM_
**Backup creation time:**  _2.16 hrs_

![image](https://user-images.githubusercontent.com/58680313/112316572-92ec5900-8cee-11eb-842c-bbf9cb45e797.png)


|Period(20:18:00 ~ 22:34:00)| All (avg) (ms) |99 Percentile (avg) (ms) | 95 Percentile (avg) (ms) | 90 Percentile (avg) (ms) | RPS (avg) (reqps) | Failure (avg) (reqps) |
|----------------------------|---------------|-------------------|--------------------|--------------------|-------------|---------------|
|23 Mar 2021 | 729 | 573  | 468 | 430 | 10 |  0 |
| 22 Mar 2021 | 637 | 503 | 419 | 383 | 12 |  0 |
| 21 Mar 2021 | 642 | 493 | 429 | 399 | 11 | 0 |

Reply:
@scalar-boney How about 20? I feel 50 sounds a bit too aggressive.

Reply:
@feeblefakie san,

> How about 20? 

### Backup creation with number of files 20 for each asynchronous upload

**Started:** _Thursday, March 25, 2021 9:50:35.769 AM_
**Finished:** _Thursday, March 25, 2021 11:18:11.817 AM_
**Backup creation time:**  _1:28 hrs_

![image](https://user-images.githubusercontent.com/58680313/112411876-f0b98900-8d60-11eb-8b35-ac0ef2b9a886.png)



|Period(09:50:00 ~ 11:18:00)| All (avg) (ms) |99 Percentile (avg) (ms) | 95 Percentile (avg) (ms) | 90 Percentile (avg) (ms) | RPS (avg) (reqps) | Failure (avg) (reqps) |
|----------------------------|---------------|-------------------|--------------------|--------------------|-------------|---------------|
|25 Mar 2021 | 1275 | 932 | 716 | 624 | 17 | 0 |
| 23 Mar 2021 | 472 | 366 | 326 | 303 | 17 |  0 |
| 21 Mar 2021 | 432 | 349 | 312 | 289 | 15 | 0 |
| 20 Mar 2021 | 458 | 377 | 331 | 302 | 18 | 0 |